tests/test_ketama.py

```diff
@@ -0,0 +1,15 @@
+from sentry_ophio.ketama import KetamaPool
+
+
+def test_hasher():
+    pool = KetamaPool(["a"])
+    assert pool.get_slot("a") == 0
+    assert pool.get_slot("b") == 0
+
+    pool = KetamaPool(["a", "b", "c", "d", "e"])
+
+    # these here are pretty random depending on the hashing state
+    assert pool.get_slot("a") == 4
+    assert pool.get_slot("b") == 3
+    assert pool.get_slot("c") == 3
```

Comment:
I rebuilt this test locally inside the twemproxy testsuite (https://github.com/twitter/twemproxy/blob/60aaf855ba2e15e20d0efbe440c8ff071b719580/src/test_all.c#L583) like this:

```c
    uint32_t idx;
    struct server_pool pool;
    struct server *server;

    memset(&pool, 0, sizeof(struct server_pool));
    pool.key_hash = hash_crc32a;
    pool.dist_type = DIST_KETAMA;

    array_init(&pool.server, 0, sizeof(struct server));
    server = array_push(&pool.server);
    server->weight = 1;
    string_copy(&server->name, "a", 1);
    server = array_push(&pool.server);
    server->weight = 1;
    string_copy(&server->name, "b", 1);
    server = array_push(&pool.server);
    server->weight = 1;
    string_copy(&server->name, "c", 1);
    server = array_push(&pool.server);
    server->weight = 1;
    string_copy(&server->name, "d", 1);
    server = array_push(&pool.server);
    server->weight = 1;
    string_copy(&server->name, "e", 1);

    ketama_update(&pool);

    idx = server_pool_idx(&pool, "a", 1);
    expect_same_uint32_t(4, idx, "a");
    idx = server_pool_idx(&pool, "b", 1);
    expect_same_uint32_t(3, idx, "b");
    idx = server_pool_idx(&pool, "c", 1);
    expect_same_uint32_t(3, idx, "c");
```